devutils/src/devutils/invoke_utils.py

```diff
@@ -380,7 +380,6 @@ def docs(ctx, check: bool = False):
         poetry(
             ctx,
             "run lazydocs",
-            "--validate",
```

Comment:
So no validation by default, until we fix **pydocstyle** and our docstring linting configuration?

Reply:
Exactly, this wasn't doing anything anyway